client/src/components/pages/jobs/Jobs.module.css

```diff
@@ -0,0 +1,11 @@
+.mainContainer {
+    padding: 2rem;
+  }
+    .header {
+    color: black;
+  }
+  .cards {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
```

Comment:
why you have the formatting like this with the indents?

don't you have in the `setting.json` this part:
```
  "[css]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
  }
```

Reply:
i added these properties